### DISCUSSION TITLE
Can I open a navbar link in a new tab?

### DISCUSSION THREAD

**Discussion by giabaio** [quarto-dev/quarto-cli#3541] [Q&A]
This is probably a question with a very simple answer, but can I enforce a navbar item to link to an external URL that opens in a new tab?

In other words, I'd like to have 
```
...
navabar: 
   right: 
      - text: "Some text here"
        href: SOME_URL
...
```
and when the user clicks on the hyperlink under `Some text here` they are taken to `SOME_URL` in a new tab. 

I don't think you can use `markdown` or pure `html` here, to say something like 
````
   - text=<a href="SOME_URL" target="_blank">Some text here</a>
````
can you?

Similarly, *without fiddling with the resulting `html` file (ie from the `_quarto.yml` directly), can I force `- icon` to use non bootstrap icons (and use instead, say, `fontawesome`)? 

I know the answer to both is probably "just replace the text in the resulting `html`, which I can do, but I wondered whether there's something more clever than that?

Thanks!

**Comment by giabaio** [comment]
I think I saw that adding `link-external-newwindow: true` in my `_quarto.yml` does the trick for most external links, but *not* for those in the navbar... Am I missing an obvious fix?
Thanks

**Comment by effigies** [comment]
I think the issue is probably that navbar links are explicitly excluded from the find-and-replace here:

https://github.com/quarto-dev/quarto-cli/blob/8c2b10dc4d25c25f413bce458b92c02eb510d6e4/src/resources/formats/html/templates/quarto-html.ejs#L388-L389

My guess is the assumption when this was written was that these were always internal links, but that no longer applies. So the fix might be as easy as removing the `not(.nav-link)` from that line.

**Comment by giabaio** [comment]
Thanks --- I think that works! (tested on a local quarto-cli installation, based on the latest release from the GitHub repo). Perhaps I should send a pull request?

**Reply by effigies** [reply]
That's what I would do.

**Comment by giabaio** [comment]
Done!

**Reply by aborruso** [reply]
Hi what is the PR? Has it been approved and is there any quarto release that contains it?

Thank you

**Reply by dragonstyle** [reply]
https://github.com/quarto-dev/quarto-cli/pull/3542

It is in 1.3 pre-release and allows Navbar items to include a `target` option.

**Reply by aborruso** [reply]
> It is in 1.3 pre-release and allows Navbar items to include a `target` option.

Thank you @dragonstyle 

I'm using `quarto-1.3.220-linux-amd64.deb` [here](https://ondata.github.io/italian-public-sector-pnrr-data-guide/rete-pnrr.html), for some sidebar menu here, but I have not target blank.
Does it work for side menu?

The [config file](https://github.com/ondata/italian-public-sector-pnrr-data-guide/blob/main/_quarto.yml#L63).

![image](https://user-images.githubusercontent.com/30607/221203780-661b38c8-8014-4b6a-b8df-be47939dca99.png)
